packages/components/pages/f-checkout/test/visual/f-checkout-authenticated.visual.desktop.spec.js

```diff
@@ -80,13 +80,11 @@ describe('f-checkout - Collection - Authenticated - Desktop Visual Tests', () =>
 
     it('should display the illegal mobile number error message', () => {
         // Arrange
-        const customerInfo = {
-            mobileNumber: '123'
-        };
+        const input = '123';
```

Comment:
I preferred the `mobileNumber` name, since it's a bit more descriptive of the data it holds.